shell/platform/darwin/macos/framework/Source/FlutterViewController_Internal.h

```diff
@@ -18,14 +21,40 @@
  */
 @property(nonatomic, readonly, nonnull) FlutterTextInputPlugin* textInputPlugin;
 
+@property(nonatomic, readonly) std::weak_ptr<flutter::AccessibilityBridgeMac> accessibilityBridge;
+
 /**
  * Returns YES if provided event is being currently redispatched by keyboard manager.
  */
 - (BOOL)isDispatchingKeyEvent:(nonnull NSEvent*)event;
 
+- (void)attachToEngine:(nonnull FlutterEngine*)engine withId:(uint64_t)viewId;
+
+- (void)detachFromEngine;
+
+/**
+ * Called by the associated FlutterEngine when FlutterEngine#semanticsEnabled
+ * has changed.
+ */
+- (void)notifySemanticsEnabledChanged;
+
+- (void)updateSemantics:(nonnull const FlutterSemanticsUpdate*)update;
```

Comment:
Does `attachToEngine`, `detachFromEngine`, and `updateSemantics` need to be documented?